### DISCUSSION TITLE
[FEATURE] Another specialization

### DISCUSSION THREAD

**Discussion by dev-master21** [azerothcore/azerothcore-wotlk#3930] [Ideas]
Hello. Can anyone suggest how you can implement another talent branch (3) without changing the client? Do it only on the server side and server core?

**Comment by BarbzYHOOL** [comment]
impossible if there is no client edit (i mean if you want an interface)

**Comment by dev-master21** [comment]
> impossible if there is no client edit (i mean if you want an interface)

No interface. In any way, so that players can, without resetting talents, pump another branch (Create creature, command)
